omod-2.0/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs2_0/ClearDbCacheController2_0Test.java

```diff
@@ -0,0 +1,168 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.webservices.rest.web.v1_0.controller.openmrs2_0;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+import org.hibernate.Query;
+import org.hibernate.SessionFactory;
+import org.hibernate.internal.CacheImpl;
+import org.junit.Test;
+import org.openmrs.Location;
+import org.openmrs.Person;
+import org.openmrs.PersonName;
+import org.openmrs.api.LocationService;
+import org.openmrs.api.PersonService;
+import org.openmrs.module.webservices.rest.web.v1_0.controller.RestControllerTestUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.mock.web.MockHttpServletResponse;
+
+public class ClearDbCacheController2_0Test extends RestControllerTestUtils {
+	
+	private static final String CLEAR_DB_CACHE_URI = "cleardbcache";
+	
+	@Autowired
+	private PersonService personService;
+	
+	@Autowired
+	private LocationService locationService;
+	
+	@Autowired
+	private SessionFactory sessionFactory;
+	
+	@Test
+	public void clearDbCache_shouldEvictTheEntityFromTheCaches() throws Exception {
+		final Class clazz = PersonName.class;
+		final Integer personNameId = 2;
```

Comment:
It would be great to follow our naming convention for constants.

Reply:
What is the convention? The naming convention I'm aware of is for **static final** constants defined at the class level

Reply:
Here it is: https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-NamingConventions